maplibre/src/vector/upload_system.rs

```diff
@@ -32,7 +33,8 @@ pub fn upload_system(
         &mut Eventually<VectorBufferPool>,
     >() else { return; };
 
-    let view_region = view_state.create_view_region();
+    let view_region =
+        view_state.create_view_region(view_state.zoom().zoom_level(DEFAULT_TILE_SIZE));
```

Comment:
this looks weird name-wise -- the `zoom_level` implies "zoom to this zoom level", but the parameter is "tile size", not some zoom.

Reply:
It is a getter. Should be renamed maybe.

Reply:
`get_zoom()` ?